custom_dc/env.list

```diff
@@ -0,0 +1,34 @@
+# API key for accessing the base Data Commons API.
+DC_API_KEY=
+# API key for accessing Maps and Places APIs for exploration tools.
+MAPS_API_KEY=
+# Use local SQLite as the database.
+USE_SQLITE=true
+# Input directory containing files to be processed.
```

Comment:
Does this need to match the "userdata" Docker workspace directory if they are running locally?  Also, how does this relate to the GCS_DATA_PATH option?

Reply:
So, does this have to be in GCS? If so, say, "Input directory in Google Cloud Storage, containing files to be processed, in the format ...."

Reply:
INPUT can be local or GCS. Added comment indicating both.